packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/ConnectPopupModal.tsx

```diff
@@ -17,18 +17,26 @@ export const ConnectPopupModal = () => {
 
     const { methodInfo, source } = popupCall;
     const { methodTitle, confirmLabel, permissionTypes } = methodInfo;
-    const { processName, origin, manifest } = source;
+
+    const rememberPayload = {
+        origin,
+        types: permissionTypes,
+        ...(source.__walletConnect
+            ? {
+                  __walletConnect: true as const,
```

Comment:
Why use underscores here?

Reply:
to signal that this is not some kind of real value but a mere branding field, I thought there was some kind of convention like this? 🤷